lsp-common.el

```diff
@@ -45,8 +45,11 @@
   "Convert Position object in PARAMS to a point."
   (save-excursion
     (goto-char (point-min))
-    (forward-line (gethash "line" params))
-    (forward-char (gethash "character" params))
+    (let ((line (gethash "line" params))
+          (char (gethash "character" params)))
+      (ignore-errors
+        (forward-line line)
+        (forward-char char)))
```

Comment:
I'd avoid `ignore-errors` if possible, it masks unrelated bugs such as type errors.
How about converting the entire function to

```elisp
  (save-restriction
    (widen)
    (+ (line-beginning-position line) column))
```

Reply:
I agree, it's a better idea

Reply:
Looks good. `xref.el` might should be updated in the same manner. It uses the following snippet:

```elisp
(save-restriction
        (widen)
        (save-excursion
          (goto-char (point-min))
          (beginning-of-line line)
          (forward-char column)
          (point-marker)))
```